docs/src/pages/components/lists/lists.md

```diff
@@ -13,22 +13,48 @@ Lists are a continuous group of text or images. They are composed of items conta
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
+## Usage Guide
```

Comment:
The heading convention: https://www.notion.so/Technical-documentation-7e55b517ac2e489a9ddb6d0f6dd765de

```suggestion
## Usage guide
```

Reply:
Also, I would recommend we start with a very basic demo, not a user guide on `ListItemSecondaryAction` that is an edge case.